### DISCUSSION TITLE
Issue with OIDC Configuration Validation

### DISCUSSION THREAD

**Discussion by davidlacho** [auth0/auth0-deploy-cli#1258] [Ideas]
### Checklist

- [X] I have looked into the [README](https://github.com/auth0/auth0-deploy-cli#readme) and have not found a suitable solution or answer.
- [X] I have looked into the [documentation](https://github.com/auth0/auth0-deploy-cli#documentation) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/auth0-deploy-cli/issues) and have not found a suitable solution or answer.
- [X] I have upgraded to the [latest version](https://github.com/auth0/auth0-deploy-cli/releases/latest) of this tool and the issue still persists.
- [X] I have searched the [Auth0 Community](https://community.auth0.com/tag/auth0-deploy-cli) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

We are migrating our Auth0 setup to our codebase using GitHub Actions. To do this, we exported the existing tenant configuration with a0deploy export --format=yaml, which produces a tenant.yaml file along with other resource configuration files. The goal is to update parts of the Auth0 configuration and then deploy the entire setup using a0deploy import --input_file ./dev/tenant.yaml.

The Issue:

Upon attempting to deploy the configuration, we encounter the following error:

Bad Request: "options.oidc_metadata.scopes_supported" must contain less than or equal to 200 items
This error suggests that our OpenID Connect (OIDC) configuration has more than 200 scopes listed in the scopes_supported field. However, this is the same configuration currently accepted and operational in our Auth0 environment, which makes this validation error puzzling.

### Expectation

This validation does not limit the number of scopes

### Reproduction

1. Given that our client's OIDC Configuration contains > 200 supported_scopes values
2. When we use our the CLI tool for deployment
3. Then the deployment fails with the above error

### Deploy CLI version

7.23.1

### Node version

v20.12.2

**Comment by kushalshit27** [comment]
Thank you for reporting this issue! Our team is currently investigating the issue.

**Comment by kushalshit27** [comment]
Apologies for the delay.  Thank you for bringing this feature request!  Currently, the client's OIDC Configuration contains 200.

To ensure this request gets the visibility it deserves and to encourage broader collaboration, we are moving this to a GitHub Discussion.
This will allow us to gather more input from the community and better prioritize this feature within our roadmap. We appreciate your input and look forward to seeing the conversation grow there!

We've already added your suggestion to our backlog for future consideration.
